index.html

```diff
@@ -27,6 +27,157 @@ <h2>I’m a software developer</h2>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+        </div>
+            </section>
+            <section class="AboutMeHeader">
```

Comment:
- [ ] Please stick to the design as much as possible, this section should be aligned in one column using flexbox

|Your design|Figma design|
|-|-|
|![Screenshot_20220531_054416](https://user-images.githubusercontent.com/25227022/171095492-7b892f7c-0cff-4d2a-b0bd-3a1d70984340.png)|![Screenshot_20220531_055239](https://user-images.githubusercontent.com/25227022/171095493-fd2cc2fb-c01c-4393-9816-fd3af49ee266.png)|

Reply:
- [ ] @AmaliaGM great work, also check on your design its missing a background image;
<img width="165" alt="image" src="https://user-images.githubusercontent.com/61408860/171218998-faaabcd4-09c6-4ffd-86fd-815933c98962.png">

Reply:
- [OPTIONAL ], could you implement these changes as the first reviewer requested.  I don't feel like this change were asked could not make you repeat, but next time implement all the changes asked by reviewers, because the following reviewer will based on that to reviewer you project.